nav2_rviz_plugins/src/nav2_panel.cpp

```diff
@@ -48,29 +54,101 @@ Nav2Panel::Nav2Panel(QWidget * parent)
   starting_->assignProperty(start_stop_button_, "text", "Shutdown");
   starting_->assignProperty(start_stop_button_, "toolTip", shutdown_msg);
 
+  // State entered after NavigateToPose has been canceled
+  canceled_ = new QState();
+  canceled_->setObjectName("canceled");
+  canceled_->assignProperty(start_stop_button_, "text", "Shutdown");
+  canceled_->assignProperty(start_stop_button_, "toolTip", shutdown_msg);
+
+  // State entered after the NavigateToPose action has completed
+  completed_ = new QState();
+  completed_->setObjectName("succesful");
+  completed_->assignProperty(start_stop_button_, "text", "Shutdown");
+  completed_->assignProperty(start_stop_button_, "toolTip", shutdown_msg);
+
+  // State entered while the NavigateToPose action is active
+  running_ = new QState();
+  running_->setObjectName("running");
+  running_->assignProperty(start_stop_button_, "text", "Cancel");
```

Comment:
The start_stop_button is not longer just start and stop, but now also cancel. Perhaps rename the variable?

Reply:
Any name suggestions? You can argue cancel is semantically a stop as well, if you don't mind the term overloading.

Reply:
Yeah, that's true. Never mind on the original comment. I think it's clear enough as is.